cms/templates/program_authoring.html

```diff
@@ -1,15 +1,16 @@
+<%page expression_filter="h"/>
 <%namespace name='static' file='static_content.html'/>
 <%! from django.utils.translation import ugettext as _ %>
 
 <%inherit file="base.html" />
 <%block name="title">${_("Program Administration")}</%block>
 
 <%block name="header_extras">
-<%static:css group='style-edx-icons'/>
 <link rel="stylesheet" href=${authoring_app_config.css_url}>
 </%block>
 
 <%block name="requirejs">
+  ## safe-lint: disable=mako-invalid-js-filter
```

Comment:
I didn't look at the error, but did you import js_escaped_string? See the read the docs page for examples.